R/gsplot-class.R

```diff
@@ -19,8 +23,12 @@ gsplot <- function(x = NULL, ...) UseMethod("gsplot")
 
 #' @rdname gsplot
 #' @export
-gsplot.default <- function(...,config.file=NA, theme=NA, frame.plot=TRUE) {
-  object <- gsplot(list(global=list('config'=list(frame.plot=frame.plot, 
+gsplot.default <- function(..., created=Sys.Date(), 
+                           gsplot.version=packageDescription("gsplot", fields = "Version"),
```

Comment:
I tried `packageVersion("gsplot")` at first, but it returns a list instead of a character string. `packageDescription("gsplot", fields = "Version")` returns a single string. To get the output of packageVersion into a single string seemed more complex than using packageDescription. 

I will make the package reference generic per @jread-usgs's suggestion.